microcosm_postgres/store.py

```diff
@@ -160,9 +160,19 @@ def search(self, *criterion, **kwargs):
 
         """
         query = self._query(*criterion)
+        query = self._order_by(query)
```

Comment:
I don't understand the usage here. Could you give an example usage of this update?